release-notes/opensearch-release-notes-2.4.0.md

```diff
@@ -0,0 +1,704 @@
+* Added legacy support for SNS messages. ([#269](https://github.com/opensearch-project/common-utils/pull/269))
+* Add list of alert ids in get alerts request  ([#284](https://github.com/opensearch-project/common-utils/pull/284))
+* Fix security-analytics alerting findings api integration ([#292](https://github.com/opensearch-project/common-utils/pull/292))
+* Added params to Datasources ([#290](https://github.com/opensearch-project/common-utils/pull/290))
+* Fix security-analytics to alerting integration ([#293](https://github.com/opensearch-project/common-utils/pull/293))
+* Add findings enabled flag and findings field in bucket level monitor ([#305](https://github.com/opensearch-project/common-utils/pull/305))
+* Support backend roles in indexMonitorRequest ([#308](https://github.com/opensearch-project/common-utils/pull/308))
+* Added function for request recreation that considers the writeable request ([#303](https://github.com/opensearch-project/common-utils/pull/303))
+* Adds owner field in monitor model ([#313](https://github.com/opensearch-project/common-utils/pull/313))
+
+
+### OpenSearch Dashboards Search Relevance
+* Add codeowner ([#43](https://github.com/opensearch-project/dashboards-search-relevance/pull/43))
+
+
+### OpenSearch Geospatial
+* Add groupId to pluginzip publication ([#167](https://github.com/opensearch-project/geospatial/pull/167))
+* Flip X and Y coordinates for WKT and array formats in XYPoint ([#156](https://github.com/opensearch-project/geospatial/pull/156))
+
+
+### OpenSearch Index Management
+* Feature/184 introduce security tests ([#474](https://github.com/opensearch-project/index-management/pull/474))
+* Alias in rollup target_index field ([#445](https://github.com/opensearch-project/index-management/pull/445))
+* Adds an alias action ([#575](https://github.com/opensearch-project/index-management/pull/575))
+* Error prevention / Action validation stage 1 ([#579](https://github.com/opensearch-project/index-management/pull/579))
+
+
+### OpenSearch Index Management Dashboards Plugin
+* feat: Add diff confirm modal in create wizard ([#323](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/323))
+* feature: Add form generator & advanced settings to speed up development in common form case ([#329](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/329))
+
+
+### OpenSearch k-NN
+* Merge efficient filtering from feature branch ([#588](https://github.com/opensearch-project/k-NN/pull/588))
+* Add groupId to pluginzip publication ([#578](https://github.com/opensearch-project/k-NN/pull/578))
+* Added sample perf-test configs for faiss-ivf, faiss-ivfpq, lucene-hnsw ([#555](https://github.com/opensearch-project/k-NN/pull/555))
+* Adding OSB index specification json for lucene hnsw ([#552](https://github.com/opensearch-project/k-NN/pull/552))
+* Adding k-NN engine stat ([#523](https://github.com/opensearch-project/k-NN/pull/523))
+
```

Comment:
```suggestion
* Add filtering in Lucene HNSW k-NN ([#519](https://github.com/opensearch-project/k-NN/issues/519)
```

Reply:
@alicejw-aws This looks like an issue but not a PR commit. @martin-gaievski Could you please help us confirm we do have such change in 2.4.0. Thanks.

Reply:
We do have it in 2.4.0 in my understanding. It is already mentioned in release note line#18.
```
You can now create custom filters to refine your k-NN searches. The filtering is tightly integrated with the Lucene HNSW algorithm and allows you to apply k-NN searches more efficiently with better search results and performance.
```

Reply:
Yes, that's correct, filtering is part of the k-NN 2.4.0